VideoLocker/res/layout/edx_divider.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<View xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="1px"
```

Comment:
Perhaps this should reference `edx_hairline` instead of hardcoding the dimension.